src/BlazorUI/Bit.BlazorUI/Components/Utilities/Image/BitImage.scss

```
@@ -1,41 +1,120 @@
-        }
-
-        100% {
-            opacity: 1;
-        }
+    // The ring is drawn on the frame rather than on the focused image: an element clips its
+    // descendants with overflow, never its own box-shadow, so a ring on the image itself would be cut
+    // off by the very frame that makes the shapes work. It is not tied to the clickable class either -
+    // an image made focusable by a TabIndex alone is reached by the same keyboard and needs the same
+    // indicator.
+    &:has(.bit-img-img:focus-visible) {
+        @include focus-ring();
```

Stylelint (17.14.0)

[error] Unexpected parentheses in argumentless mixin "focus-ring" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)